cache/src/commonMain/kotlin/org/mobilenativefoundation/store/cache5/CacheBuilder.kt

```diff
@@ -2,7 +2,7 @@ package org.mobilenativefoundation.store.cache5
 
 import kotlin.time.Duration
 
-class CacheBuilder<Key : Any, CommonRepresentation : Any> {
+class CacheBuilder<Key : Any, Common : Any> {
```

Comment:
how do you feel about renaming common to output? As in it is the output type

Reply:
My concern with `Output` was `write()` and `Converter` would be taking in `Output`. I think having one name across all read and write helps with Store mental model. So I think we should either:
1. Rename every `Common` to `Output` and rename `input` parameters to `value`
2. Use `Common` throughout with `input` parameters

My vote is for 2 but I'm open to 1

Reply:
Changed a few files - `value` and `Output` reads just as well. Will change the rest 👍🏽

Reply:
And yeah I do like having `Network`, `Local`, and `Output` overall